package.json

```diff
@@ -61,6 +62,7 @@
     "i18next-conv": "^5.0.0",
     "i18next-express-middleware": "^1.0.2",
     "i18next-sprintf-postprocessor": "^0.2.2",
+    "if-env": "^1.0.0",
```

Comment:
Has to be in non dev dependencies so it is available after `install --production` however on the good side the library is very tiny.